Sources/WordPressShared/WordPressShared.swift

```diff
@@ -0,0 +1,5 @@
+#if SWIFT_PACKAGE
+
+@_exported import WordPressSharedObjC
+
+#endif /* SWIFT_PACKAGE */
```

Comment:
Nitpick, given there's currently only one `#if`:

```suggestion
#endif
```